woocommerce/woocommerce-android
# [HACK] Fix whats new dialog size on big screens

Closes: #12676 


### Description

The PR makes whats new dialog look ok on the bigger screens

### Steps to reproduce

I was using code modification, but naturally, this should be reproducible via updating an older app to a new version
[Simulate_Announcement_pop_up.patch](https://github.com/user-attachments/files/17129430/Simulate_Announcement_pop_up.patch)

### Testing information

* Try different screen sizes in both landscape and portrait orientation
* Notice that it looks ok now

### The tests that have been performed

The once above

### Images/gif

<img width="405" alt="image" src="https://github.com/user-attachments/assets/58f14ee2-8d9b-42ca-82ef-8d625342e053">
<img width="627" alt="image" src="https://github.com/user-attachments/assets/c7add4ae-efb5-4b4a-b197-94f3ebf7e64b">
<img width="459" alt="image" src="https://github.com/user-attachments/assets/20ffd7c2-d0fa-4b82-a81b-c198610d2821">
<img width="638" alt="image" src="https://github.com/user-attachments/assets/b6471b10-dffc-44cd-98a9-5a94639a8136">
<img width="811" alt="image" src="https://github.com/user-attachments/assets/6452f012-575d-484f-a569-33f92186a465">
<img width="618" alt="image" src="https://github.com/user-attachments/assets/44e72586-93d2-4cfd-9c56-f78126e09739">



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
